### PR TITLE
Text mode selecting improvement

### PR DESCRIPTION
**Pull request by Dmitry-VF**
### Description of Change ###

RNz83G1JaE

### Issues Resolved ### 

- fixes #

### API Changes ###

 
None

### Platforms Affected ### 

- All
- WPF
- UWP
- iOS
- Android

### Behavioral Changes ###

None

### PR Checklist ###

- [ ] I have included examples or tests
- [ ] I have updated the change log
- [ ] I am listed in the CONTRIBUTORS file
- [ ] Rebased on top of the target branch at time of PR
- [ ] Changes adhere to coding standard

### CODE REVIEW

File: src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml [hunk 1]
```diff
@@ -59,6 +77,20 @@
...
+                <Style.Triggers>
+                    <Trigger Property="Text" Value="{StaticResource Empty}">
+                        <Setter Property="BorderBrush" Value="{StaticResource DottedBrush}" />
+                    </Trigger>
+                    <DataTrigger Binding="{Binding RelativeSource={RelativeSource AncestorType={x:Type controls:DateTimePicker}}, 
+                        Path=IsMouseOver}" Value="True">
+                        <Setter Property="BorderThickness" Value="1" />
+                    </DataTrigger>
+                    <Trigger Property="IsMouseOver" Value="True">
+                        <Setter Property="Background" Value="{DynamicResource AccentColorBrush5}" />
```
Comment (ceo1647) [inline src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml:91]: use
{DynamicResource Orc.Brushes.Control.MouseOver.Background} instead

File: src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml [hunk 2]
```diff
@@ -46,6 +49,20 @@
                 </Setter>
             </Style>
 
+            <DrawingBrush x:Key="CheckersBrush" Viewport="0,0,0.25,0.25" TileMode="Tile">
+                <DrawingBrush.Drawing>
+                    <DrawingGroup>
+                        <GeometryDrawing Brush="White">
```
Comment (GeertvanHorrik) [inline src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml:55]: Shouldn't white depend on the current theme?

File: src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml [hunk 3]
```diff
@@ -59,6 +76,24 @@
...
+                <Style.Triggers>
+                    <MultiDataTrigger>
+                        <MultiDataTrigger.Conditions>
+                            <Condition Binding="{Binding Path=Text, RelativeSource={RelativeSource Self}}" Value="{StaticResource Empty}"/>
+                            <Condition Binding="{Binding RelativeSource={RelativeSource AncestorType={x:Type controls:DateTimePicker}}, Path=IsMouseOver}" Value="True"/>
+                        </MultiDataTrigger.Conditions>
+
+                        <Setter Property="Background" Value="{StaticResource CheckersBrush}"/>
+                        <Setter Property="BorderThickness" Value="0 0 0 2"/>
+                        <Setter Property="BorderBrush" Value="LightGray"/>
```
Comment (GeertvanHorrik) [inline src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml:90]: @ceo1647 , don't we have special gray brushes in orc.theming for this as well?

Comment (ceo1647) [reply src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml:90]: checkers brush is another static resource that depends on our theming
image

File: src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml (hunk 2 above)
Comment (ceo1647) [reply src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml:55]: forgot, thanks

Comment (ceo1647) [reply src/Orc.Controls/Controls/DateTimePicker/Themes/DateTimePicker.generic.xaml]: fixed

**Review by GeertvanHorrik — APPROVED** [review]

**Pull request merged**